src/lib/browse/MrnLayer.svelte

```diff
@@ -54,6 +55,13 @@
           Major Road Network
         </ExternalLink>.
       </p>
+      <p>
```

Comment:
Going back to the stand-up this morning, some kind of standardized data catalog where we record license, owner, when we accessed the data, etc would be handy, and we can surface some of this info if it's useful